codalab/bundles/run_bundle.py

```diff
@@ -29,8 +29,10 @@ class RunBundle(DerivedBundle):
                                                                     'codalab/default-cpu:latest) we wish to use.', completer=DockerImagesCompleter, hide_when_anonymous=True, default=None))
     METADATA_SPECS.append(MetadataSpec('request_time', str, 'Amount of time (e.g., 3, 3m, 3h, 3d) allowed for this run. Defaults to user time quota left.', formatting='duration', default=None))
     METADATA_SPECS.append(MetadataSpec('request_memory', str, 'Amount of memory (e.g., 3, 3k, 3m, 3g, 3t) allowed for this run.', formatting='size', default='2g'))
+    METADATA_SPECS.append(MetadataSpec('memory_usage_out_of_limit', str, 'Portion of memory used by this container (e.g., 93.31MiB / 13.67GiB, 8.035MiB / 13.67GiB)', completer=DockerStatsCompleter))
```

Comment:
For consistency with `cpu_usage`:

```suggestion
    METADATA_SPECS.append(MetadataSpec('memory_usage', str, 'Portion of memory used by this container (e.g., 93.31MiB / 13.67GiB, 8.035MiB / 13.67GiB)', completer=DockerStatsCompleter))
```